reference/6/Microsoft.PowerShell.Management/Start-Process.md

```diff
@@ -128,6 +128,25 @@ Start-Process -FilePath "$env:comspec" -ArgumentList "/c dir `"%systemdrive%\pro
 Start-Process -FilePath "$env:comspec" -ArgumentList "/c","dir","`"%systemdrive%\program files`""
 ```
 
+### Example 8: Create a detached process on Linux
+
+On Windows, `Start-Process` creates an independent process in a new console window that remains
```

Comment:
I'm not sure I'd say "new console window".  -NoNewWindow can prevent the new window and since it's just a process, it may not be a console at all.

Reply:
Fixed